### ISSUE TITLE
Is this ready for use in production?

### ISSUE THREAD

**Issue by cloud-tribal** [vue-bulma/vue-admin#140]
Just curious to know whether this is currently being used in production. API looks amazing

**Comment by luventa** [comment]
I have a small project, which has almost the same structure with vue-admin-1.0, deployed to production in the company I'm working for. And it works good.

There are at lease two items we need to work on if you want use vue-admin in a real webapp
1. Communication functionality with a back-end server. You can use vue-resource and vuex which are already included to implement the functionality.
2. This project does not contain any business related logic.

**Comment by cloud-tribal** [comment]
Where do you see 1.0?

**Comment by luventa** [comment]
[vue-admin 1](https://github.com/vue-bulma/vue-admin/tree/vue-v1)

Version 1 is based on Vue 1.0.X

**Comment by Gonzalo2683** [comment]
A question,
What do I need to upload to the server in order to see the vue-amdin demo working?

I run the npm run build command and generated several files, I tried to test it on local with apache server, but it gives me an error like this every time I navigate to a different section:
[Vue-router] Failed to resolve async component Default: Error: Loading chunk 33 failed.

As I can have the demo run on the local, other than the npm run dev option

In short, what are the files I need to upload to the server to have the demo working?

Thanks
